apps/addresses/models.py

```diff
@@ -65,6 +65,25 @@ def clone(self):
         )
         return address_clone
 
+    def json_encode(self):
+        json_encode = {
```

Comment:
please rename the variable to something like `payload` or `data`

Reply:
Done